### PR TITLE
Update CreateHtmlEmailSignature.scpt to work with newer MacOS's

### PR DESCRIPTION
**Pull request by benthomasian**
Updating the script to work with MacOS 10.10 and 10.11, ignore the last Pull request please and try this one? I commented out lines that were needed to make this work.

### CODE REVIEW

**Comment by seesolve** [comment]
Thanks for the patch. I can't merge it because it takes out what makes the script work for earlier versions.

**Comment by benthomasian** [comment]
Would it make sense to start a new version for MacOS 10.10+ or do we bother to try to modify the original code?  Also, to solve the existing signature issue, can we set the script to find the current date and time on the local machine so when it creates the "Company Standard"signature it makes sure that it's using that one and not one that already exists?

**Comment by seesolve** [comment]
Whether we start a new version or we modify the original code, the code is going to need more than a superficial rewrite (based on what I'm seeing on the step-by-step instructions at the link you provided). 

The script already looks for the most recently created signature to edit.

And there are more problems to script for in the new versions than that. Reading the instructions at the link you provided, is step 2 necessary? If you simply create a signature but do not associate it with any email address and quit Mail, is the signature still there when you restart Mail?

**Comment by benthomasian** [comment]
Yes, the file is still there **IF** you lock the file.

**Comment by seesolve** [comment]
Ugh. Well, I think locking the file is going to be easier to script than trying to drag and drop the signature to an email account.

File: CreateHtmlEmailSignature.scpt [hunk 1]
```diff
@@ -29,8 +29,8 @@ end if
 
 set textOSVersion to system version of (system info)
 
-if ((textOSVersion does not start with "10.7") and (textOSVersion does not start with "10.8")) then
-	display dialog "You are using OS" & space & textOSVersion & ". This script is configured to run on 10.7 or 10.8 and will now quit." buttons {textButtonOkay} default button textButtonOkay cancel button textButtonOkay with title "Script Quitting"
+if ((textOSVersion does not start with "10.10") and (textOSVersion does not start with "10.11")) then
```
Comment (seesolve) [inline CreateHtmlEmailSignature.scpt]: We want to maintain the previous code while incorporating changes so the script will work with newer versions.

File: CreateHtmlEmailSignature.scpt [hunk 2]
```diff
@@ -29,8 +29,8 @@ end if
...
+	display dialog "You are using OS" & space & textOSVersion & ". This script is configured to run on 10.10 or 10.11 and will now quit." buttons {textButtonOkay} default button textButtonOkay cancel button textButtonOkay with title "Script Quitting"
```
Comment (seesolve) [inline CreateHtmlEmailSignature.scpt]: Same thing. We want to maintain the previous code while incorporating changes so the script will work with newer versions.

File: CreateHtmlEmailSignature.scpt [hunk 3]
```diff
@@ -40,9 +40,6 @@ else
 end if
 
 
-my subCheckGuiScriptingEnabled()
```
Comment (seesolve) [inline CreateHtmlEmailSignature.scpt]: Same thing. We want to maintain the previous code while incorporating changes so the script will work with newer versions.

File: CreateHtmlEmailSignature.scpt [hunk 4]
```diff
@@ -103,7 +100,7 @@ end repeat
 
 --
 -- Create or modify mail signature file: 
--- for 10.7, file is created; for 10.8, file is modified
+-- for 10.10, file is created; for 10.11, file is modified
```
Comment (seesolve) [inline CreateHtmlEmailSignature.scpt]: I think that's inaccurate (that 10.10 will have the file created).

File: CreateHtmlEmailSignature.scpt [hunk 5]
```diff
@@ -180,26 +177,6 @@ end subTestForInternetConnection
 
 
 --
--- Handler to enable GUI scripting
```
Comment (seesolve) [inline CreateHtmlEmailSignature.scpt]: Same thing. We want to maintain the previous code while incorporating changes so the script will work with newer versions.

File: CreateHtmlEmailSignature.scpt [hunk 6]
```diff
@@ -216,7 +193,7 @@ on subMailSignaturesPrefsOpen()
 			
 			-- add new signature
 			tell window 1
-				click button "Signatures" of tool bar 1
+				click button "Signatures" of toolbar 1
```
Comment (seesolve) [inline CreateHtmlEmailSignature.scpt]: Has the UI element changed from tool bar to toolbar?

**Pull request closed without merging**